### PR TITLE
AVX-36508: update Edge BGP over WAN underlay

### PR DESCRIPTION
**Pull request by shuoliugit**

### CODE REVIEW

File: goaviatrix/transit_external_device_conn.go [hunk 1]
```diff
@@ -59,6 +59,7 @@ type ExternalDeviceConn struct {
 	AuthType               string `form:"auth_type,omitempty"`
 	EnableEdgeUnderlay     bool   `form:"edge_underlay,omitempty"`
 	RemoteCloudType        string `form:"remote_cloud_type,omitempty"`
+	BgpMd5KeyChanged       bool   `form:"bgp_md5_key_changed,omitempty"`
```
Comment (zhiqiangjin) [inline goaviatrix/transit_external_device_conn.go:62]: Why is this needed?

**Review by zhiqiangjin — CHANGES_REQUESTED** [review]

File: goaviatrix/transit_external_device_conn.go (hunk 1 above)
Comment (shuoliugit) [reply goaviatrix/transit_external_device_conn.go:62]: > Why is this needed?

it's special for edge

**Review by zhiqiangjin — APPROVED** [review]
